hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestObjectStoreV1.java

```diff
@@ -264,6 +268,95 @@ public void testLookupKey() throws Exception {
             dirPathC.getObjectID(), true);
   }
 
+  @Test
+  public void testRenameKey() throws IOException {
+    String fromKeyName = UUID.randomUUID().toString();
+    String value = "sample value";
+    OzoneClient client = cluster.getClient();
+
+    ObjectStore objectStore = client.getObjectStore();
+    OzoneVolume volume = objectStore.getVolume(volumeName);
+    OzoneBucket bucket = volume.getBucket(bucketName);
+    createTestKey(bucket, fromKeyName, value);
+
+    // Rename to empty string should fail.
+    OMException oe = null;
+    String toKeyName = "";
+    try {
+      bucket.renameKey(fromKeyName, toKeyName);
+      fail("Rename to empty string should fail!");
+    } catch (OMException e) {
+      oe = e;
+    }
+    Assert.assertEquals(OMException.ResultCodes.INVALID_KEY_NAME,
+            oe.getResult());
```

Comment:
Can we simplified all this check to below? I didn't see _oe_ is used in other place.
```java
    try {
      bucket.renameKey(fromKeyName, toKeyName);
      fail("Rename to empty string should fail!");
    } catch (OMException e) {
      Assert.assertEquals(OMException.ResultCodes.INVALID_KEY_NAME,
            e.getResult());
    }
```

Reply:
Agreed!